src/docs/arch-index.dox

```diff
@@ -127,6 +127,10 @@ We go into some detail for some of the internal components.
 
 The Block Manager manages the reading and writing of disk blocks.
 
+@subpage arch-backup
+
+The backup is a type of cursor used to backup the database.
```

Comment:
I'd reword as:
'Hot backup uses a type of cursor to backup the database.'

Reply:
fixed as suggested.